modules/gathering-clusterversion-history.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history_{context}"]
+= Gathering ClusterVersion history 
+
+When updating your cluster, the cluster version operator (CVO) creates a `status.history` entry to record the update. To gather and understand the history of your successful or failed cluster update, it is important to retrieve the ClusterVersion history.
+
+[NOTE]
+====
+The ClusterVersion history has a size limit, so in cases where enough updates occurred to have reached that limit, the least relevant entries in the history are pruned to accommodate the limit. 
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. View a detailed summary of cluster specifications, update availability, and update history:
```

Comment:
nit: looking at `.status.history` just gives you "update history".  It would be `availableUpdates` for availability, and... something under `spec` for cluster specifications?  "detailed summary" also seems like it's pulling in two directions, and `history` seems more on the detailed side of things and less on the summary side of things.  Maybe something like:

> View cluster update history:

?